src/com/lilithsthrone/game/character/body/tags/FaceTypeTag.java

```diff
@@ -0,0 +1,21 @@
+package com.lilithsthrone.game.character.body.tags;
+
+/**
+ * @since 0.3.6.1
+ * @version 0.3.6.1
+ * @author Stadler76
+ */
+public enum FaceTypeTag {
+	/** Standard aka human-ish faces. This is the default and can be left out. */
+	STANDARD,
+	/** Muzzled faces */
+	MUZZLE,
+	/** Fangs instead of normal teeth */
+	FANGS,
+	/** Beaked faces */
+	BEAK,
+	/** Pointy, trianguler shark teeth (Currently NYI, but planned for the (far?) future) */
+	SHARK_TEETH,
+	/** Being bald aka hairLength == 0 is not unnatural for that faceType */
+	NATURAL_BALDNESS,
+}
```

Comment:
I could add some parser commands to be used in conditionals for that.
Suggestions:
- `npc.hasMuzzle`
- `npc.hasFangs`
- `npc.hasBeak`
- `npc.hasSharkTeeth`
- `npc.isFaceBaldnessNatural`, `npc.isBaldnessNatural`, `npc.isFaceNaturallyBald` and `npc.isNaturallyBald`

Reply:
I'll get those added after merging it; good idea! ^^

Reply:
(I've added them as methods in the GameCharacter class, which can be called in parsing conditionals.)